java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -830,6 +830,7 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
                 .xgroupCreate(
                         streamKey1, groupName2, "0-0", StreamGroupOptions.builder().makeStream().build())
                 .xgroupCreateConsumer(streamKey1, groupName1, consumer1)
+                .xgroupSetId(streamKey1, groupName1, "0-2")
```

Comment:
Please add `xgroupSetId` with 4 arguments in this test along with the version check. See other tests (line 735) for example.

Reply:
Done.